src/Spago/Build.hs

```diff
@@ -323,15 +342,22 @@ runBackend
 runBackend maybeBackend RunDirectories{ sourceDir, executeDir } moduleName maybeSuccessMessage failureMessage extraArgs = do
   logDebug $ display $ "Running with backend: " <> fromMaybe "nodejs" maybeBackend
   BuildOptions{ pursArgs } <- view (the @BuildOptions)
-  isES <- Purs.hasMinPursVersion "0.15.0"
-  let postBuild = maybe (nodeAction isES $ Path.getOutputPath pursArgs) backendAction maybeBackend
-  build (Just postBuild)
+  isES <- Purs.hasMinPursVersion "0.15.0-alpha-01"
+  nodeEsSupport <- hasNodeEsSupport
+  case (isES, nodeEsSupport) of
+    (True, Unsupported nv) -> die [ "Unsupported Node.js version: " <> display (Version.prettySemVer nv), "Required Node.js version >=12." ]
```

Comment:
Usecase: we might be running an alternate backend and not have node installed in the first place. Here we'd kill the program, while we were supposed to keep running!
So this check should go inside the `nodeAction`.

Reply:
Fixed in latest commits. However, I'd guess the fix grants too many capabilities to `build`'s `postBuild` action. I'm not familiar with all the different capabilities, so I've just gone with a simple fix for now.

Reply:
That's alright, thanks!